logout.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <button>logout</button>
```

Comment:
```suggestion
   <button type="submit">logout</button> 
```
ditambahkan type nya agar tombol nya mempunyai fungsi submit